core/src/main/java/org/springframework/security/access/expression/method/DefaultMethodSecurityExpressionHandler.java

```diff
@@ -109,12 +109,14 @@ private MethodSecurityExpressionOperations createSecurityExpressionRoot(Supplier
 	}
 
 	/**
```

Comment:
Will you please change the copyright date on any edited files that are `20xx-20yy` to be `20xx-2024`?